08week/tic-tac-toe/script.js

```diff
@@ -1,5 +1,60 @@
 'use strict';
 
 $(document).ready(function() {
-  // Put app logic in here
+  var clickCount= 0;
+  $('[data-cell]').click(function(){
+    clickCount= clickCount+1;
+    var whichDiv=$(this).attr('data-cell');
+    // console.log(whichDiv);
+    var move=$(this).text();
+    // console.log(move);
+// getting the X to work
+    if ((clickCount % 2 === 0) && (move === "")) {
+      // console.log("getting the X to work");
+      $(this).text("X");
+      // getting the O to work
+    } else if ((clickCount % 2 !== 0) && (move === "")) {
+      // console.log("getting the O to work");
+      $(this).text("O");
+      // if none of those conditions are met, you can't play that move
+    } else {
+      alert("That square isn't empty. Try again in an empty square.");
```

Comment:
Since you are using click count to determine if an X or an O should be played, click count should not increment if an non-empty square is clicked.